config/routes.js

```diff
@@ -179,6 +179,8 @@ module.exports.routes = {
   'get /api/searchpackages' : 'SearchController.packageSearch',
   'get /api/searchfunctions' : 'SearchController.functionSearch',
   'get /search_packages' : 'SearchController.packageSearch',
-  'get /search_functions' : 'SearchController.functionSearch'
+  'get /search_functions' : 'SearchController.functionSearch',
 
+  // Parser
+  'get /parser/list': 'ParserController.list',
```

Comment:
What you are getting back is json and consumable by other services, so I would put the endpoint under `/api`, something like `/api/packages/to-parse`